docs/components/navigation.css

```diff
@@ -1,28 +1,37 @@
-.navigation {
+.navigation ul {
+  list-style-type: none;
+  padding-inline-start: 0px;
 }
 
-.navigation h4 {
-  margin-left: 10px;
+.navigation ul:not(.depth0) {
+  padding-inline-start: 8px;
```

Comment:
This is pretty experimental (I'd never heard of it!) and specifically Edge wouldn't understand it (https://developer.mozilla.org/en-US/docs/Web/CSS/padding-inline-start#Browser_compatibility).

Can we change it to something more normal?